### PR TITLE
Node tests setup

### PR DESCRIPTION
**Pull request by SMakaiTakori**
Adding in Node testing

### CODE REVIEW

File: ci/tests.sh
Comment (jeffmaher) [inline ci/tests.sh:7]:
```suggestion
```

This can be removed.

File: ci/tests.sh
Comment (jeffmaher) [inline ci/tests.sh:8]:
```suggestion
npx playwright install
```

I don't know if this hurts, but tried removing the --with-deps and it worked OK.

File: playwright.config.js
Comment (jeffmaher) [inline playwright.config.js:24]:
```suggestion
  reporter: 'line',
```

The previous setting (`html`) causes it to run a server that hosts an HTML report. This means that the Playwright command will hang since it requires user input to quit the test run. `line` just reports out what happened on the command line and terminates when the test run is complete.

Comment (SMakaiTakori) [reply playwright.config.js:24]: Ah, okay, that makes sense!

**Comment by SMakaiTakori** [comment]
@jeffmaher @LynnHerrick9 Our tests are officially working! I tested it with both passing and failing tests and it's working as expected so I think we are good to merge this in once you both look it over and approve. Thanks team!

File: Dockerfile
Comment (LynnHerrick9) [inline Dockerfile:15]: Small request -- can you add a comment here about why we switched to the bullseye version and when/if we should consider updating

**Review by LynnHerrick9 — COMMENTED** [review]
Overall looks good.  Could you add a very short section to the README for what should be run by developers prior to making PR (so they can be sure PR step will pass) and potentially a good place to start if the tests fail (this is optional, if you feel like errors given in console aren't helpful)?

File: Dockerfile
Comment (SMakaiTakori) [reply Dockerfile:15]: Yes, that would be a good place for it!

File: ci/tests.sh
Comment (SMakaiTakori) [reply ci/tests.sh:7]: Okay, when I did it originally, it didn't work but I the `html` to `line` change could have been what was causing the hang up. Going to remove the deps and see what it does.

**Review by LynnHerrick9 — APPROVED** [review]

**Review by jeffmaher — APPROVED** [review]

**Pull request merged**
